categories.py

```diff
@@ -151,11 +148,11 @@
     "forthommel": ["proton-pog"],
     "cardinia": ["tau-pog"],
     "mbluj": ["tau-pog"],
-    "kskovpen": ["pdmv", "tracking-pog"],
+    "kskovpen": ["tracking-pog"],
     "mmasciov": ["tracking-pog"],
     # PPD
     "malbouis": ["ppd"],
-    "jordan-martins": ["ppd"],
+    "vlimant": ["ppd"],
```

Comment:
@vlimant , can you please remove this duplicate entry ( it is already there at https://github.com/cms-sw/cms-bot/blob/master/categories.py#L71 )